kms/dstack-app/deploy-to-vmm.sh

```diff
@@ -32,9 +32,15 @@ else
 # The URL of the dstack app image download URL
 # IMAGE_DOWNLOAD_URL=https://files.kvin.wang/images/mr_{OS_IMAGE_HASH}.tar.gz
 
+# Image hash verification feature flag
+# VERIFY_IMAGE=true
```

Comment:
Let's uncomment them for backward compatibility.
```suggestion
VERIFY_IMAGE=true
```